docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,15 @@
 ### What was the problem?
 
-### How did I fix it?
+### How did I solve?
 
-### How to test it?
+### How to manually test it?
 
 ### Review checklist
 
-* The PR resolves #INSERT_ISSUE_NUMBER
-* All new code is covered with unit tests
-* All new code was formatted with Prettier
-* Linting passes
-* Tests pass
-* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
-* Documentation has been added/updated
+- The PR resolves #INSERT_ISSUE_NUMBER
+- All new code is covered with unit tests
+- All new code was formatted with Prettier
+- Linting passes
+- Tests pass
+- Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
+- Documentation has been added/updated
```

Comment:
Updating documentation in PR is not a common case. Maybe we should have separate issues labeled as docs instead? Also maybe "Updated JSDoc comments" is a better requirement for us?

Reply:
Some additional rules:

```
[ ] - Synced with target branch
[ ] - Added to relevant Project
[ ] - Added to relevant Milestone
[ ] - Connected to issue on Zenhub
```

Reply:
```
[ ] - "pending review" label was assigned
```

Reply:
this is general template, so i don't think adding those will be good.
- JSDocs doesn't apply to elements and commander
- target branch is enforced by the github setting
- relevant project, milestone, zenhub doesn't apply to general contributors